src/rpc/blockchain.cpp

```diff
@@ -1292,7 +1292,7 @@ UniValue DeploymentInfo(const CBlockIndex* blockindex, const ChainstateManager&
 }
 } // anon namespace
 
-static RPCHelpMan getdeploymentinfo()
```

Comment:
Why the `RPCHelpMan`  is not `static` anymore?

Reply:
@Riahiamirreza To be able to use it in `src/rest.cpp`, see `getblockchaininfo`.